ui/src/components/Listing/index.tsx

```diff
@@ -58,14 +59,58 @@ interface ListingProps {
   isFetching?: boolean;
   items: ListingItemProps[];
   Actions?: React.FC<{ id: string }>;
+  handlePagination?: any;
 }
 
+interface PaginationProps {
+  isNext: boolean;
+  isPrevious: boolean;
+  handleOffset: any;
```

Comment:
Please refrain from using `any`. The name can be improved as well. It is near impossible for me to tell from this line exactly what this function is doing.

Reply:
Reading code below, `onChangePagination`, or `onPaginate` sounds better names.

Reply:
Type could be: `onPaginate: (direction: 'previous' | 'next') => void`